mrchecker-app-under-test/src/main/java/com/capgemini/mrchecker/selenium/pages/projectY/StatusCodesCodePage.java

```diff
@@ -41,17 +41,34 @@ public String getCodeNumber() {
 		return codeNumber;
 	}
 	
+	/**
+	 * Verifies if page is loaded with given code number.
+	 * 
+	 * @param codeNumber
+	 *            Expected code number
+	 * @return true if expected code number is loaded with web page
+	 */
 	public boolean isLoadedWithStatusCode(String codeNumber) {
 		return getDriver().getCurrentUrl()
 						.equals(GetEnvironmentParam.THE_INTERNET_MAIN_PAGE.getValue() + PageSubURLsProjectYEnum.STATUS_CODES.getValue() + "/" + codeNumber);
 	}
 	
+	/**
+	 * Returns displayed code number.
+	 * 
+	 * @return String object representing displayed code number
+	 */
 	public String getDisplayedCodeNumber() {
-		String wholeText = getDriver().findElementDynamic(selectorDisplayedText)
-						.getText();
-		return wholeText.substring(21, 24); // Get from text "This page returned a 200 status code." status code value
+		return getDriver().findElementDynamic(selectorDisplayedText)
+						.getText()
+						.substring(21, 24);
```

Comment:
Could you please add to Javadoc information about these 'magic' numbers?

Reply:
Javadoc updated.